build.sh

```diff
@@ -35,12 +35,15 @@ integration=0
 platform=""
 rails_env="production"
 rails_env_file=""
-while getopts "hptr:c:m:" options; do
+no_cache=""
```

Comment:
I'm going to revert this back since it looks like @sk274 has actually implemented something similar with his PR: https://github.com/cul-it/blacklight-cornell/pull/2325